### DISCUSSION TITLE
Deleted GitHub Files Keep Coming Back

### DISCUSSION THREAD

**Discussion by planglands3** [community/community#73338] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How do we prevent deleted files from coming back. I deleted a bunch of files and committed it but the files reappear the next day and I have to delete them again. What do I do? Please help.

**Comment by charlesritchea** [comment]
Did you push after committing? And they are coming back after you do what? git pull or git pull --rebase etc.?

**Reply by planglands3** [reply]
Yes I did a push and the files are still coming back. I read something about wiping out the GitHub history as the files may be cached somewhere. I am not sure what is going on.

**Reply by charlesritchea** [reply]
_when_ are the files coming back? How is your workspace getting updated? An IDE plugin? or CLI git pull etc.?

**Reply by planglands3** [reply]
Files are coming back randomly. I delete, push, then maybe they will come back 5 days from now. I wonder if they deleted files are saved somewhere. I am using GitHub Desktop.

**Reply by charlesritchea** [reply]
I've never used GitHub Desktop. I'm assuming it's like an IDE where there are per repository settings. See if there are setting to sync your workspace
